message EntityMessage{
	required string methodName = 1;
	required string paramDict 	= 2;
}